examples/stac-item.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: example.html
+title: STAC support
+shortdesc: Rendering a STAC Item with geometry and COG.
+docs: >
+  This example uses the npm package [`ol-stac`](https://m-mohr.github.io/ol-stac) which
```

Comment:
```suggestion
  This example uses the npm package [`ol-stac`](https://m-mohr.github.io/ol-stac/) which
```

Minor, but this change avoids the unnecessary redirect.